static/lang/re-en.json

```diff
@@ -437,7 +437,9 @@
         },
         "Mutagens": {
             "BestialMutagen": {
-                "FeralPrompt": "Increase penalty to AC to increase damage dice size?"
+                "FeralPrompt": "Increase penalty to AC to increase damage dice size?",
+                "FeralNo": "No",
+                "FeralYes": "Yes"
```

Comment:
These are too generic to go in here. Honestly they can go live in the actual ChoiceSet localization strings `PF2E.UI.RuleElements.ChoiceSet.Yes`